internal/infrastructure/persistence/sqlite/queries/history.sql

```
@@ -36,26 +43,54 @@ DELETE FROM history WHERE id = ?;
+SELECT
+    date(last_visited) as day,
+    COUNT(*) as entries,
+    SUM(visit_count) as visits
+FROM history
+WHERE last_visited >= date('now', ?)
+GROUP BY day
+ORDER BY day ASC;
 
 -- name: DeleteHistoryByDomain :exec
-DELETE FROM history WHERE url LIKE '%://' || ? || '/%' OR url LIKE '%://' || ? || '?%' OR url LIKE '%://' || ? || '#%' OR url LIKE '%://' || ?;
+DELETE FROM history WHERE domain = @domain;
```

SQLFluff (4.1.0)

[error] The 'WHERE' keyword should always start a new line.

(LT14)